generators/entity/files.js

```diff
@@ -213,7 +213,7 @@ const angularFiles = {
             templates: [
                 {
                     file: 'entities/_entity-management.component.html', method: 'copyHtml', template: true,
-                    renameTo: generator => `entities/${generator.entityFolderName}/${generator.entityFileName}.component.html`
+                    renameTo: generator => `entities/${generator.entityFolderName}/${generator.entityPluralFileName}.component.html`
```

Comment:
@VictorADS @flaviencathala I'm ok with the PR so once you change the file name here you can merge this